acme/src/main/java/io/micronaut/acme/events/CertificateEvent.java

```diff
@@ -22,27 +22,42 @@
  * Event used to alert when a new ACME certificate is ready for use.
  */
 public class CertificateEvent {
-    private final X509Certificate certificate;
     private final KeyPair domainKeyPair;
+    private final X509Certificate[] fullCertificateChain;
     private boolean validationCert;
 
     /**
+     * @deprecated See constructor that takes full certificate chain instead.
```

Comment:
I think you can use something like this in the javadoc: 

`@deprecated {@link #CertificateEvent(KeyPai, boolean,X509Certificate)} instead.`

Reply:
Fixed here [84db7da](https://github.com/micronaut-projects/micronaut-acme/pull/161/commits/84db7da34769c2c6423db67454f9482ef2b59c4b)